databroker/broker.py

```diff
@@ -884,3 +902,38 @@ def _munge_time(t, timezone):
     """
     t = datetime.fromtimestamp(t)
     return timezone.localize(t).replace(microsecond=0).isoformat()
+
+
+class HeaderSourceShim(object):
```

Comment:
This is here for circular import reasons.  Once the refatoring/rotating is done, the re-arrangement will be easier.